src/TraceEvent/Ctf/CtfMetadataTypes.cs

```diff
@@ -51,7 +51,7 @@ public static int AlignDown(int val, int alignment)
             return val & ~(alignment - 1);
         }
 
-        public static long AlignDown(long val, long alignment)
+        public static long AlignDown(long val, int alignment)
```

Comment:
This seems fine, but just curious - what kind of issue did you run into here?

Reply:
No issue. Just noticed the discrepancy (rest is using `int` as alignment and it makes sense) when working there.